app/views/business_categories/show.html.haml

```diff
@@ -1,15 +1,28 @@
 %header.entry-header
-  %h1.entry-title= 'Företagstyp: ' "#{@business_category.name}"
+  %h1.entry-title
+    Hundföretag av typen
+    = @business_category.name
 .post-title-divider
   %span
 .entry-content
-  %p
-    %b Namn:
-    = @business_category.name
-  %p
-    %b Beskrivning:
-    = @business_category.description
-.center
-  = link_to 'Redigera', edit_business_category_path(@business_category)
-  \|
-  = link_to 'Alla kategorier', business_categories_path
+  - unless @business_category.description.nil?
+    %p
+      %b Beskrivning:
+      = @business_category.description
+  %table
+    %thead
+      %tr
+        %th Namn
+        %th Län
+        - if current_user.try(:admin)
+          %th Org nr
+          %th
+    %tbody
+      - @business_category.companies.each do |company|
```

Comment:
You may want to get unique companies within category (because we are going through membership_applications to get companies, and one company could occur in more than one membership_application).

Also, may want to sort the companies by name (not a big deal until we have a lot of companies, of course).

Something like:
```
@business_category.companies.distinct.sort {|cp1, cp2| cp1.name <=> cp2.name}.each do |company|
```